internal/generator/vector/output/cloudwatch/cloudwatch_test.go

```diff
@@ -122,20 +122,20 @@ var _ = Describe("Generating vector config for cloudwatch output", func() {
 			conf := New(outputSpec.Name, outputSpec, []string{"cw-forward"}, secrets, nil, op)
 			Expect(string(exp)).To(EqualConfigFrom(conf))
 		},
-			Entry("when groupName is spec'd", "{{.log_type}}-foo", func(spec *obs.OutputSpec) {},
+			Entry("when groupName is spec'd", `{.log_type||"missing"}-foo`, func(spec *obs.OutputSpec) {},
 				framework.NoOptions, "cw_with_groupname.toml"),
-			Entry("when URL is spec'd", "{{.log_type}}", func(spec *obs.OutputSpec) {
+			Entry("when URL is spec'd", `{.log_type||"missing"}`, func(spec *obs.OutputSpec) {
 				spec.Cloudwatch.URL = "http://mylogreceiver"
 			}, framework.NoOptions, "cw_with_url.toml"),
-			Entry("when minTLS and ciphers is spec'd", "{{.log_type}}", nil, testhelpers.FrameworkOptionWithDefaultTLSCiphers, "cw_with_tls_and_default_mintls_ciphers.toml"),
-			Entry("when tls is spec'd", "{{.log_type}}", func(spec *obs.OutputSpec) {
+			Entry("when minTLS and ciphers is spec'd", `{.log_type||"missing"}`, nil, testhelpers.FrameworkOptionWithDefaultTLSCiphers, "cw_with_tls_and_default_mintls_ciphers.toml"),
```

Comment:
How are TLS and ciphers relevent here?

Reply:
I didn't add these tests, I only converted them to obs api as well as now with the groupname changes. I can remove them if they aren't relevant